tests/execution-spec-tests/get_execution_spec_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
```

Comment:
```suggestion
#!/bin/bash
set -euxo pipefail
```

This set of options makes it so that it will fail the whole script in any part fails.

Reply:
Additionally, I removed any `echo`s in favor of the `x` trace option.